src/main/java/org/cloudfoundry/promregator/lite/discovery/DiscoveryControllerV2.kt

```diff
@@ -0,0 +1,55 @@
+package org.cloudfoundry.promregator.lite.discovery
+
+import com.fasterxml.jackson.annotation.JsonGetter
+import com.fasterxml.jackson.annotation.JsonInclude
+import kotlinx.coroutines.runBlocking
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import javax.servlet.http.HttpServletRequest
+
+data class DiscoveryV2Response(
+        val targets: List<String>,
+        val labels: DiscoveryLabelV2,
+)
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+class DiscoveryLabelV2(
```

Comment:
Kotlin equivalent of the existing `DiscoveryLabel` class. Allows for Kotlin object creation techniques/default values.